CHANGELOG.md

```diff
@@ -45,6 +46,47 @@ RELEASE CHANGELOG TEMPLATE:
 ### Documentation
 -->
 
+<a name="v2.3.0"></a>
+# [2.3.0](https://github.com/ligato/vpp-agent/compare/v2.2.0...v2.3.0) (2019-10-04)
+
+### Compatibility
+- **VPP 20.01** (rc0)
```

Comment:
Define exact version that can be installed from packagecloud: `20.01-rc0~161-ge5948fb49~b3570`

```
➤ docker run --rm -it ligato/vpp-base:20.01-rc0.161-ge5948fb49 cat /vpp/version
20.01-rc0~161-ge5948fb49~b3570
```